admin/assets/stylesheets/elements/_flash-messages.scss

```diff
@@ -15,6 +15,22 @@
   }
 }
 
+.info-message-clickable {
+  border: 5px solid $green;
+  padding: 1em;
+  margin-top: 40px;
+  color: $green;
+  font-weight: bold;
+  text-decoration: underline;
+  a,
+  a:link,
+  a:visited,
+  a:active,
+  a:hover {
+    color: $green;
+  }
+}
+
```

Comment:
Hmmmm, does this have to be explicitly underlined? I thought `.info-message` div texts will be already underlined when there’s a link inside them. If there's text (not a link) that's underlined, won't that be confusing to the users?

...if that should have the exact info-message properties with the addition that it has to be explicitly underlined, maybe an inner `&.clickable { text-decoration: underline; color: $green; }` would be better than duplicating the props (and have the `<div class="info-message clickable">`)? 🤔

Reply:
@danwilliams good catch I tried to do that but it did not work for some weird reason and I ended up duplicating which indeed is ugly.
I updated it just now and it works like a charm.
Many thanks!